app/components/pagy_paginator_component.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class PagyPaginatorComponent < ApplicationComponent
+  include PagyHelper
+  renders_one :loading_icon
+
+  attr_reader :id, :pagy, :url_array, :container_classes, :view_component
+
+  def initialize(id:, pagy:, url_array:, container_classes:, direction: :bottom)
+    @id = id
+    @pagy = pagy
+    @url_array = url_array
```

Comment:
I wasn't sure what to name this parameter. I wanted to imply you should feed it an array (it's easiest to just merge the page param into the url if it's an array). I could see wanting the option to feed it a string and then writing logic to add the page param with either a `?` or `&` depending on what it detect. Either way, just wanted to point this out

Reply:
I'd rather pass in the symbol (e.g. `developer`) and infer the array from there. Then we can use that to automatically se the `id` without needing to pass it in as well. Appending `-pagy`, like you are already doing, should be enough so there's no collision.

Reply:
inferring the array seemed less flexible to me. If that's not a concern for this app, that's totally fine. I was just anticipating nested or namespaces resources, as I have ran into this before. Just let me know how you'd like it changed and I'll take care of it.

Reply:
Got it, we can keep it as is for now then. Thanks!